src/libraries/System.Runtime.Experimental/src/System.Runtime.Experimental.proj

```diff
@@ -1,7 +0,0 @@
-<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
I guess this was causing a no-op incremental build of the ref?

Reply:
No it wasn't. This project was required in the past when we kicked off packaging via a glob filter: `src/libraries/*/src/*.*proj`. Now that packaging happens implicitly via the `GeneratePackageOnBuild` property, this project isn't required anymore.